cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaBridgeConsumerConfiguration.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2017-2018, Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+
+package io.strimzi.operator.cluster.model;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+
+
+/**
+ * Class for handling Kafka Bridge consumer configuration passed by the user
+ */
+public class KafkaBridgeConsumerConfiguration extends AbstractConfiguration {
```

Comment:
I think we should configure here and in the producer configuration options the forbidden options such as `ssl.`, `sasl.`, `security.protocol`, `bootstrap.servers`.